packages/storage/src/bucket.js

```diff
@@ -1032,6 +1135,41 @@ Bucket.prototype.makePublic = function(options, callback) {
 };
 
 /**
+ * Set labels on the bucket.
+ *
+ * This makes an underlying call to {module:storage/bucket#setMetadata}, which
+ * is a PATCH request. This means an individual label can be overwritten, but
+ * unmentioned labels will not be touched.
+ *
+ * @param {type} labels - Labels to set on the bucket.
+ * @param {function} callback - The callback function.
+ * @param {?error} callback.err - An error returned while making this request.
+ * @param {object} callback.metadata - The bucket's metadata.
+ *
+ * @example
```

Comment:
To be consistent with the unit test below, maybe we should use

var labels = {
   labelone: 'labelonevalue',
   labeltwo: 'labeltwovalue'
};